### PR TITLE
HPCC-15540 ESDL should provide command to unbind method from ESDL Binding

### PR DESCRIPTION
**Pull request by rpastrana**
Signed-off-by: rpastrana rodrigo.pastrana@lexisnexis.com

### CODE REVIEW

**Comment by rpastrana** [comment]
@wangkx please review.

File: tools/esdlcmd/esdl-publish.cpp [hunk 1]
```diff
@@ -958,6 +958,183 @@ class EsdlBindMethodCmd : public EsdlBindServiceCmd
...
+        else
+            fprintf(stdout, "\n Received empty configuration for ESDL Binding %s.\n", bindingid.str());
+
+        return success;
+    }
+
+    void usage()
+    {
+        printf( "\nUsage:\n\n"
+                "esdl unbind-method <TargetESPProcessName> <TargetESPBindingName> <TargetServiceName> <TargetServiceDefVersion> <TargetMethodName> [\n\n"
```
Comment (wangkx) [inline tools/esdlcmd/esdl-publish.cpp]: Should the <TargetServiceDefVersion>be here?

File: tools/esdlcmd/esdl-publish.cpp [hunk 2]
```diff
@@ -958,6 +958,183 @@ class EsdlBindMethodCmd : public EsdlBindServiceCmd
...
+                    break;
+                case 1:
+                    optBindingName.set(arg);
+                    break;
+                case 2:
+                    optService.set(arg);
+                    break;
+                //case 3:
+                //    optVersionStr.set(arg);
+                //    break;
```
Comment (wangkx) [inline tools/esdlcmd/esdl-publish.cpp]: Remove the 3 lines

**Comment by wangkx** [comment]
@rpastrana looks fine. 2 minor comments.

**Comment by rpastrana** [comment]
@wangkx please review latest changes.

**Review by wangkx — APPROVED** [review]
Looks fine.

**Comment by rpastrana** [comment]
@richardkchapman please merge.

**Pull request merged**